lib/http-outgoing.js

```diff
@@ -120,6 +122,8 @@ export default class PodletClientHttpOutgoing extends PassThrough {
         this.#manifest = {
             // @ts-expect-error Internal property
             _fallback: '',
+            _js: [],
+            _css: [],
```

Comment:
Cache properties for js and css.
These get (re)populated any time the fallback route is fetched